tests/Resources/Page/OCPDashboard/InstalledOperators/InstalledOperators.robot

```diff
@@ -40,6 +40,7 @@ Search Installed Operator
 
 Is Operator Installed
   [Arguments]  ${operator}
+  Run Keyword and Return Status     Wait Until Element is Visible      //a[@data-test-operator-row="${operator}"]    timeout=10
```

Comment:
isn't it redundant with the next line where you get the ${is_installed} variable?

Reply:
it just a time taken  for operator to be appear. When we search the operator it take sometime to appear that's the reason i added for wait where as is installed fetch web element and return status. I tried to avoid usage of sleep

Reply:
okay so maybe you don't need the next lines. If this line succeeds, you have the "install" status of the operator. Correct?

Reply:
yeah we actually do not need that..i have kept it as extra precautions nothing more